src/fiatconnect/index.ts

```diff
@@ -144,16 +144,14 @@ export async function getFiatConnectQuotes(
     provider: fiatConnectProviders.find((provider) => provider.id === result.id)!,
   }))
 }
-export type FetchQuotesInput = QuotesInput & {
-  fiatConnectCashInEnabled: boolean
-  fiatConnectCashOutEnabled: boolean
-}
+export type FetchQuotesInput = QuotesInput
 
 export async function fetchQuotes(params: FetchQuotesInput) {
-  const { fiatConnectCashInEnabled, fiatConnectCashOutEnabled, ...quotesInput } = params
-  if (!fiatConnectCashInEnabled && params.flow === CICOFlow.CashIn) return []
```

Comment:
we still need the logic for returning empty array when `params.flow === CICOFlow.CashIn` right?